edx_lint/tamper_evident.py

```diff
@@ -26,21 +26,21 @@ def write(self, text, hashline=u"# {}"):
         be changed to accommodate different file syntaxes.
 
         Arguments:
-            text (unicode string): the contents of the file to write.
+            text (byte string): the contents of the file to write.
 
-            hashline (unicode string): the format of the last line to append to
+            hashline (byte string): the format of the last line to append to
                 the file, with "{}" replaced with the hash.
```

Comment:
It's really stupid that the only reason we care about which encoding is because bytestrings don't have a .format() method... :(   In any case, requiring UTF8 is fine with me.